data-products-dp0-3/index.rst

```diff
@@ -74,18 +74,18 @@ Catalogs
 --------
 
 .. list-table:: Catalog data available for DP0.3.
-   :widths: 100 390
+   :widths: 20 80
```

Comment:
I'm not sure why the width ratio was changed, but this doesn't accommodate the new text in the first column. The formatting, once rendered, is not very readable.

![Screenshot 2024-11-06 at 11 25 36 AM](https://github.com/user-attachments/assets/be929648-95cf-4ce6-a20b-b677706bdff9)

Would recommend either to try a width ratio more like 2:3 than 1:4, or perhaps create a new second column for the links?

Reply:
improved appearance of the table